features/step_definitions/authorization.js

```diff
@@ -37,4 +46,82 @@ module.exports = function server() {
             return null;
         });
     });
+
+    this.Then(/^they can see the project$/, () =>
+        request({                           // make sure pipeline exists
```

Comment:
I wasn't sure how to create a pipeline any earlier without the user's jwt.
After discussing with @FenrirUnbound , we decided it would make sense to do it in the _Given an existing pipeline with that repository_ in the future. We will need to make the call to get a jwt with the admin's **accessKey** before creating a pipeline.

Since we don't have any other users yet, we will wait to do this refactoring.